src/Symfony/Component/HttpClient/Response/AmpResponse.php

```diff
@@ -216,7 +195,7 @@ private static function select(ClientState $multi, float $timeout): int
         return null === self::$delay ? 1 : 0;
```

Comment:
There shouldn't be any manual calls to `Loop::run()` and `Loop::stop()` anymore.

Reply:
yes, but I'm struggling a bit with figuring out the correct replacement :)
something with suspension?

Reply:
Likely suspensions, yes. Suspensions shouldn't be cached in a static property here, as they're fiber-specific. So if this is called from two different fibers, it'll break. Use a local variable instead.